packages/backend/package.json

```diff
@@ -47,7 +47,7 @@
     "@backstage/plugin-search-backend-node": "1.2.17",
     "@internal/plugin-dynamic-plugins-info-backend": "*",
     "@internal/plugin-scalprum-backend": "*",
-    "@janus-idp/backstage-plugin-rbac-backend": "2.4.1",
+    "@janus-idp/backstage-plugin-rbac-backend": "2.5.0",
```

Comment:
2.6.1 is the latest version, do you want to update https://www.npmjs.com/package/@janus-idp/backstage-plugin-rbac-backend?activeTab=versions? 

cc @AndrienkoAleksandr @PatAKnight